src/NuGet.Core/NuGet.Configuration/Settings/Items/PackageSourceNamespacesItem.cs

```diff
@@ -0,0 +1,233 @@
+// Copyright(c) .NET Foundation.All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Xml.Linq;
+using NuGet.Shared;
+
+namespace NuGet.Configuration
+{
+    /// <summary>
+    /// A PackageSourceNamespacesItem has only a key and at least 1 <see cref="NamespaceItem"/> child item.
+    ///     - [Required] Key
```

Comment:
Can we use [list syntax ](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/list#syntax) in xml doc?

```suggestion
    /// <list type="bullet">
    /// <item>
    /// <description>[Required] Key</description>
    /// </item>
    /// </list>
```

Reply:
I was really just trying to be consistent with the rest of the code comments. 

See: 

https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Configuration/Settings/Items/CredentialsItem.cs
https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Configuration/Settings/Items/FileClientCertItem.cs

Reply:
I like when IntelliSense formats documentation comments, specially when using xmldoc syntax. 

I didn't realize there was a previous convention. It's fine to keep current convention.